apps/api/services/zpe/enroll.py

```
@@ -0,0 +1,106 @@
+
+    def consume_token(
+        self,
+        token: str,
+        *,
+        name: Optional[str] = None,
+        meta: Optional[Dict[str, str]] = None,
+    ) -> ComputeServerRegistration:
+        key = f"{_ENROLL_PREFIX}{token}"
+        deleted = cast(int, self.redis.delete(key))
+        if deleted == 0:
+            raise KeyError("token not found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)